app/cypress/fixtures/secureHrefTests.js

```diff
@@ -5,80 +5,107 @@ export default [
   {
     title: '# href (valid)',
     valid: true,
+    url: '#example.com',
     content: {
       document: {
         ...minimalDoc.document,
-        acknowledgments: [{ urls: ['#test.org'] }],
+        acknowledgments: [{ urls: ['#example.com'] }],
       },
     },
   },
   {
     title: 'http href (valid)',
     valid: true,
+    url: 'http://example.com',
     content: {
       document: {
         ...minimalDoc.document,
-        acknowledgments: [{ urls: ['http://test.org'] }],
+        acknowledgments: [{ urls: ['http://example.com'] }],
       },
     },
   },
   {
     title: 'https href (valid)',
     valid: true,
+    url: 'https://example.com',
     content: {
       document: {
         ...minimalDoc.document,
-        acknowledgments: [{ urls: ['https://test.org'] }],
+        acknowledgments: [{ urls: ['https://example.com'] }],
       },
     },
   },
   {
     title: 'mailto href (valid)',
     valid: true,
+    url: 'mailto://user@example.com',
     content: {
       document: {
         ...minimalDoc.document,
-        acknowledgments: [{ urls: ['mailto://user@test.org'] }],
+        acknowledgments: [{ urls: ['mailto://user@example.com'] }],
       },
     },
   },
   {
     title: 'tel href (valid)',
     valid: true,
+    url: 'tel://\\+4934587913248',
     content: {
       document: {
         ...minimalDoc.document,
-        acknowledgments: [{ urls: ['tel://+4934587913248test.org'] }],
+        acknowledgments: [{ urls: ['tel://+4934587913248'] }],
```

Comment:
We can't use a real number here. I checked with the authority, we can use any of those described in https://www.bundesnetzagentur.de/SharedDocs/Downloads/DE/Sachgebiete/Telekommunikation/Unternehmen_Institutionen/Nummerierung/Rufnummern/mittlg148_2021.pdf?__blob=publicationFile&v=1

=> +493023125000